pkg/upload/js.go

```diff
@@ -16,9 +24,163 @@ type JsOptions struct {
 	Path        utils.Paths `arg:"" name:"path" help:"Path to a directory of source maps and bundles to upload" type:"path" default:"."`
 }
 
+// Resolve the project if it isn't specified using the currend working directory
+func resolveProjectRoot(projectRoot string, path string) string {
+	if projectRoot != "" {
+		return projectRoot
+	}
+	workingDirectory, err := os.Getwd()
+	if err != nil {
+		return path
+	}
+	return workingDirectory
+}
+
+// Attempt to parse information from the package.json file if values aren't provided on the command line
+func ResolveVersion(versionName string, projectRoot string, logger log.Logger) (string, error) {
+	if versionName != "" {
+		return versionName, nil
+	}
+	checkPath, err := filepath.Abs(projectRoot)
+	if err != nil {
+		return "", fmt.Errorf("unable to make project root an absolute path %s", projectRoot)
```

Comment:
This should be a warning too - if it doesn't exist, we just don't find the package.json.